node-api/backend/genesis.go

```diff
@@ -35,3 +35,31 @@ func (b Backend) GenesisValidatorsRoot(slot math.Slot) (common.Root, error) {
 	}
 	return st.GetGenesisValidatorsRoot()
```

Comment:
This function should be redone.
- does not need to take in a slot
- should only read from state if the value is not already cached.
- if not cached, call stateFromSlot(0) and explicitly comment that we are reading from the state at the tip of the chain.
- cache and return st.GetGenesisValidatorsRoot()

This is okay since we actually store the genesis value (and remains unmodified) in the app-enforced state.

Reply:
addressed in [commit](https://github.com/berachain/beacon-kit/pull/2601/commits/1c56d57d0e7b4e35deac6135bd840dfa764a5933)